doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -316,6 +316,10 @@ Cellular samples
 
   * Added the configuration overlay file :file:`overlay-supl.conf` for building the sample with SUPL assistance support.
 
+* :ref:`udp` sample:
+
+  * Added the :kconfig:option:`CONFIG_UDP_DATA_UPLOAD_ITERATIONS` Kconfig option for configuring the number of data transmissions to the server.
```

Comment:
```suggestion
  * Added the :ref:`CONFIG_UDP_DATA_UPLOAD_ITERATIONS <CONFIG_UDP_DATA_UPLOAD_ITERATIONS>` Kconfig option for configuring the number of data transmissions to the server.
```

Reply:
@divipillai Is this the new standard syntax to use from now on?

Reply:
This is the way to represent the application-specific Kconfig options. For example, for UDP sample, the ones that are defined here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/cellular/udp/README.html#configuration. The Kconfig options defined in the Kconfig reference (library-specific) are provided with the ":kconfig:option:" syntax.